### ISSUE TITLE
Add language support for Hebrew

### ISSUE THREAD

**Issue by YovelB** [logisim-evolution/logisim-evolution#1739]
A few colleges and universities use the tool in Israel, mainly in Electronics Engineering.

I would to add support for the Hebrew language 
Do I have permission to do so?

**Comment by maehne** [comment]
Of course! Such contributions are welcome! Just create a PR against the branch `develop`. If you want early feedback on your work, you can mark the PR as a draft and already push an incomplete state, which you then complete with additional commits.

**Comment by maehne** [comment]
One challenge, which might require some investigation from your side is how to handle the different writing direction.

**Comment by YovelB** [comment]
Thanks for the feedback @crossplatformdev. When I tried dealing with this issue I had problems understanding swing and the project base code. So, I have abandoned it, without making it clear (my fault).

If someone volunteers to deal with this issue it would be great. I would be happy to help in the locale and formatting side. Else, feel free to close the issue.

**Comment by crossplatformdev** [comment]
Hi @maehne ,

What about simply writing the strings reversed in the properties Hebrew locale strings file?

Will be very simple workaround, just in case the user needs it.

Kind regards,
Elías A.

**Comment by YovelB** [comment]
I dont see this as important enough, closing issue.
